include/protozero/basic_pbf_builder.hpp

```diff
@@ -67,7 +67,7 @@ class basic_pbf_builder : public basic_pbf_writer<TBuffer> {
      * @param tag Tag of the field that will be written
      */
     template <typename P>
-    basic_pbf_builder(basic_pbf_writer<TBuffer>& parent_writer, P tag) noexcept :
+    basic_pbf_builder(basic_pbf_writer<TBuffer>& parent_writer, P tag) :
```

Comment:
We're removing noexcept here, presumably because it was never actually noexcept in the first place?

Reply:
Exactly. It was never noexcept, clang-tidy found that problem. We can not make it noexcept, so we have to remove the `noexcept`.